x-pack/plugins/security_solution/public/common/components/filter_group/index.tsx

```diff
@@ -249,7 +256,7 @@ const FilterGroupComponent = (props: PropsWithChildren<FilterGroupProps>) => {
           title: item.title,
           selectedOptions: item.selectedOptions ?? [],
           existsSelected: item.existsSelected ?? false,
-          exclude: item.existsSelected,
```

Comment:
Typo that caused bug - [Security Solution] Alerts filters are automatically changed #155481